File: celciusToFahrenheit.cpp
#include <iostream>
using namespace std;

void celciusToFahrenheit(int celcius){
	double Fahrenheit = (celcius * 1.8000) + 32.00;
	cout << Fahrenheit << "°F" << endl;
}

int main(){
	int celcius;
	cin >> celcius;
	celciusToFahrenheit(celcius);
}

File: celciusToFahrenheit.java
import java.util.Scanner;
public class Main {
	
	public static void celciusToFahrenheit(int celcius){
		double Fahrenheit = (celcius * 1.8000) + 32.00;
		System.out.println(Fahrenheit + "°F");
	}
	
	public static void main(String[] args) {
		Scanner in = new Scanner(System.in);
		int celcius;
		celcius = in.nextInt();
		
		celciusToFahrenheit(celcius);
	}
}
